examples/jsm/nodes/materials/NodeMaterial.js

```diff
@@ -89,6 +90,12 @@ class NodeMaterial extends ShaderMaterial {
 
 			outputNode = this.constructOutput( builder, vec4( outgoingLightNode, diffuseColor.a ) );
 
+			// OUTPUT NODE
+
+			builder.stack.assign( output, outputNode );
```

Comment:
So `output` is not defined in the case of unlit materials?

Reply:
Yes, because if `unlit` is true so `material.outputNode` is equal `output`.